### PR TITLE
fix(flux): add durations to Flux logging (#19697)

### PR DESCRIPTION
**Pull request by lesam**
(cherry picked from commit 25fb1077e2dd5469e7a3055db76fc71a99e95139)

Closes #

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Feature flagged (if modified API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: flux/control/controller.go [hunk 1]
```diff
@@ -37,22 +91,44 @@ type Controller struct {
 }
 
 func (c *Controller) Query(ctx context.Context, compiler flux.Compiler) (flux.Query, error) {
+	requestStart := time.Now()
 	for _, dep := range c.deps {
 		ctx = dep.Inject(ctx)
 	}
 
+<<<<<<< HEAD
```
Comment (davidby-influx) [inline flux/control/controller.go:99]: This merge failure looks unhappy!

File: flux/control/controller.go [hunk 2]
```diff
@@ -37,22 +91,44 @@ type Controller struct {
...
 	p, err := compiler.Compile(ctx, runtime.Default)
+||||||| parent of 25fb1077e2... fix(flux): add durations to Flux logging (#19697)
+	p, err := compiler.Compile(ctx)
+=======
```
Comment (davidby-influx) [inline flux/control/controller.go:103]: Merge failure?

File: flux/control/controller.go [hunk 3]
```diff
@@ -37,22 +91,44 @@ type Controller struct {
...
 	}
 
+	alloc := &memory.Allocator{}
+	return p.Start(ctx, alloc)
+}
+
+func (c *Controller) compile(ctx context.Context, compiler flux.Compiler, requestStart time.Time) (flux.Program, error) {
+	start := time.Now()
+	p, err := compiler.Compile(ctx)
+>>>>>>> 25fb1077e2... fix(flux): add durations to Flux logging (#19697)
```
Comment (davidby-influx) [inline flux/control/controller.go:116]: Merge failure?

**Review by davidby-influx — CHANGES_REQUESTED** [review]
Looks like some errors, noted in the comments.

**Comment by lesam** [comment]
Oops, missed a merge conflict. Fixed now.

**Comment by lesam** [comment]
And fixed a stale test too

**Review by davidby-influx — APPROVED** [review]
LGTM

**Pull request merged**